_layouts/home.html

```diff
@@ -0,0 +1,227 @@
+                    <li><a href="#what" class="btn btn-white">Learn More</a></li>
+                </ul>
+            </div>
+            <div class="col-lg-5 col-md-6 mr-auto">
+                <div class="inner-img">
+                    <img src="{{ '/assets/img/intro.svg' | relative_url }}">
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="container text-center why-vac" id="why">
```

Comment:
Can we comment this out for now? It doesn't resonate with the motivation of the _protocol_ effort, but more for something like the Status app.

E.g. see https://vac.dev/vac-overview

Reply:
Commented